Makefile

```diff
@@ -12,6 +12,15 @@ test:
 	make shellcheck
 	make docker-unit-test
 
+unit-test-heroku-20:
+	@docker run -v $(PWD):/project danielleadams/shpec:latest
```

Comment:
FWIW: You can run single CircleCI jobs on your machine with their [CLI](https://circleci.com/docs/2.0/local-cli/#run-a-job-in-a-container-on-your-machine). This would ensure that local unit tests are run exactly the same way as on CircleCI with less duplicate config to maintain. It would add a new dependency for users of the Makefile though. 

I plan to move all local test execution to the CircleCI CLI and get rid of `Makefile`s for local test execution. This is what the docs look like: https://github.com/heroku/heroku-buildpack-jvm-common#run-tests-locally

Reply:
Oh nice! Yes, this is something to consider for the future once we iron out the integration to CircleCI. Will make note of it for when @danielleadams gets back (: